src/restruct_poc/model/node.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "src/restruct_poc/types.hpp"
+
+namespace openturbine {
+
+struct Node {
+    int ID;      // Node identifier
+    Array_7 x;   // Node positions and orientations
+    Array_7 u;   // Node displacement
+    Array_6 v;   // Node velocity
+    Array_6 vd;  // Node acceleration
```

Comment:
This design gives me a pause -- should this be packaged into another data structure and composed to `Node`?

Reply:
Possibly. `State` contains about the same set of data, except for `x`.